contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -112,11 +104,14 @@ contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions
    *  - adminFee: fee that will be paid to the Router owner for operating the network
    */
   function _updateConfig(bytes memory config) internal override {
-    (uint96 adminFee, bytes4 handleOracleFulfillmentSelector, uint32[] memory maxCallbackGasLimits) = abi.decode(
-      config,
-      (uint96, bytes4, uint32[])
-    );
+    (
+      uint16 maxConsumers,
```

Comment:
should we call it "maxConsumersPerSubscription"?

Reply:
I like that. How about I change this in the next coming PR so that I can get through the merge queue?